winternlc/create/make_linearpoly_corrections.py

```diff
@@ -0,0 +1,293 @@
+import os
+import sys
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+import numpy as np
+from utils import extract_pixel_values, find_median_files, get_exposure_time
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../")))
+from config import DEFAULT_CUTOFF, output_directory, test_directory
+from numpy.polynomial import Chebyshev
+
+LINEAR_REGION_MINCOUNTS = 20000
+LINEAR_REGION_MAXCOUNTS = 50000
+SATURATION = 56000
+PLOT = False
```

Comment:
If this is actually an arguments (`PLOT`) you should just put it in the function itself. When we import this in mirar we can't edit the scripts themselves to toggle this on/off.